omod/src/test/java/org/openmrs/module/idgen/web/resource/LogEntryResourceTest.java

```diff
@@ -0,0 +1,71 @@
+/* * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.idgen.web.resource;
+
+import java.util.Date;
+import org.junit.Before;
+import org.openmrs.User;
+import org.openmrs.api.UserService;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.idgen.IdentifierSource;
+import org.openmrs.module.idgen.LogEntry;
+import org.openmrs.module.idgen.rest.resource.LogEntryResource;
+import org.openmrs.module.idgen.service.IdentifierSourceService;
+import org.openmrs.module.webservices.rest.web.ConversionUtil;
+import org.openmrs.module.webservices.rest.web.resource.impl.BaseDelegatingResourceTest;
+
+public class LogEntryResourceTest extends BaseDelegatingResourceTest<LogEntryResource, LogEntry> {
+    @Before
+    public void setup() throws Exception {
+        executeDataSet("org/openmrs/module/idgen/include/TestData.xml");
+    }
+
+    @Override
+    public String getDisplayProperty() {
+        return getObject().getSource().toString();
+    }
+
+    @Override
+    public String getUuidProperty() {
+        return getObject().getIdentifier();
+    }
+
+    @Override
+    public LogEntry newObject() {
+        UserService service = Context.getUserService();
+        IdentifierSource logSource = Context.getService(IdentifierSourceService.class).getIdentifierSourceByUuid("0d47284f-9e9b-4a81-a88b-8bb42bc0a901");
+        Date dateGenerated = (Date) ConversionUtil.convert("2017-10-03 12:00:00.433", Date.class);
+        User generatedBy = service.getUserByUuid("1010d442-e134-11de-babe-001e378eb67e");
+        return new LogEntry(logSource, "100892", dateGenerated, generatedBy, "New patient");
+    }
+
+    @Override
+    public void validateDefaultRepresentation() throws Exception {
```

Comment:
Did you just forget to validate the ref representation?

Reply:
I have added the ref representation validation